.gitignore

```diff
@@ -1,3 +1,5 @@
+*.*ignore
```

Comment:
Why do we need this?

Reply:
Someone can use ignore files that are specific to something but that are not needed for others. So this will ignore those.

Reply:
Can you give a solid example

Reply:
`.prettierignore` used when formatting with `Prettier`